src/Core/Tools/Authorizer/PostAuthorizer.php

```diff
@@ -80,7 +80,11 @@ public function isAllowed(Entity $entity, $privilege)
         }
 
         // First check whether there is a role with the right permissions
-        if ($this->acl->hasPermission($user, Permission::MANAGE_POSTS)) {
+        if (($privilege != "delete") && ($this->acl->hasPermission($user, Permission::MANAGE_POSTS))) {
```

Comment:
Let's use strict comparison here. 
```suggestion
        if (($privilege !== "delete") && ($this->acl->hasPermission($user, Permission::MANAGE_POSTS))) {
```

Reply:
thanks @kubabialy